src/photos/targets/public/App.jsx

```diff
@@ -134,6 +122,9 @@ class App extends Component {
 const ConnectedApp = props => (
   <Query query={ALBUM_QUERY} {...props}>
     {({ data: album, fetchStatus }) => {
+      if (fetchStatus === 'failed') {
```

Comment:
maybe I missed something but I think you can remove `state` in the dumb component since its `fetchStatus` can't be `failed`